jetbrains-core/tst/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SendMessagePaneTest.kt

```diff
@@ -0,0 +1,172 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.sqs.toolwindow
+
+import com.intellij.ide.util.PropertiesComponent
+import com.nhaarman.mockitokotlin2.whenever
+import kotlinx.coroutines.runBlocking
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito
+import software.amazon.awssdk.services.sqs.SqsClient
+import software.amazon.awssdk.services.sqs.model.SendMessageRequest
+import software.amazon.awssdk.services.sqs.model.SendMessageResponse
+import software.aws.toolkits.core.region.AwsRegion
+import software.aws.toolkits.core.utils.RuleUtils
+import software.aws.toolkits.jetbrains.core.region.MockRegionProvider
+import software.aws.toolkits.jetbrains.services.sqs.MAX_LENGTH_OF_FIFO_ID
+import software.aws.toolkits.jetbrains.services.sqs.Queue
+import software.aws.toolkits.jetbrains.utils.BaseCoroutineTest
+import software.aws.toolkits.jetbrains.utils.waitForTrue
+import software.aws.toolkits.resources.message
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class SendMessagePaneTest : BaseCoroutineTest() {
+    private lateinit var client: SqsClient
+    private lateinit var region: AwsRegion
+    private lateinit var standardQueue: Queue
+    private lateinit var fifoQueue: Queue
+    private lateinit var standardPane: SendMessagePane
+    private lateinit var fifoPane: SendMessagePane
+    private lateinit var cache: PropertiesComponent
+
+    @Before
+    fun reset() {
+        client = mockClientManagerRule.create()
+        region = MockRegionProvider.getInstance().defaultRegion()
+        standardQueue = Queue("https://sqs.us-east-1.amazonaws.com/123456789012/standard", region)
+        fifoQueue = Queue("https://sqs.us-east-1.amazonaws.com/123456789012/fifo.fifo", region)
+        cache = PropertiesComponent.getInstance(projectRule.project)
+        standardPane = SendMessagePane(client, standardQueue, cache)
+        fifoPane = SendMessagePane(client, fifoQueue, cache)
+    }
+
+    @Test
+    fun `No input fails to send for standard`() {
+        standardPane.apply {
+            inputText.text = ""
+            runBlocking { sendMessage() }
+        }
+
+        assertTrue { standardPane.bodyErrorLabel.isVisible }
```

Comment:
Ah I didn't notice this before, but we don't use kotlin test asserts (I need to add that to the import ktlint rule), we use the asssertJ versions